File: mobile-client/ios/Contract/Catalog/Detail/CatalogNodeDetailPayload.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
public struct CatalogNodeDetailPayload: Sendable, Equatable {
    public let node: CatalogNodeSummary
    public let description: String?
    public let breadcrumbLabels: [String]
    public let featuredProductIds: [String]
}

File: mobile-client/android/Contract/Catalog/Detail/CatalogNodeDetailPayload.kt
package app.mobiling.client.client.contract.catalog.detail

import app.mobiling.client.client.contract.catalog.browse.CatalogNodeSummary

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class CatalogNodeDetailPayload(
    val node: CatalogNodeSummary,
    val description: String?,
    val breadcrumbLabels: List<String>,
    val featuredProductIds: List<String>,
)
